javascript-local-storage/data.js

```diff
@@ -0,0 +1,11 @@
+/* exported todos */
+
+let todos = [];
+
+const previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```

Comment:
This should be inside an `if` statement making sure `previousTodosJSON` does not equal `null`